examples/seismic/model.py

```diff
@@ -397,3 +397,105 @@ def critical_dt(self):
         # for further details:
         dt = .85*np.min(self.spacing) / (np.sqrt(self.grid.dim)*self.maxvp)
         return self.dtype("%.3e" % dt)
+
+class ModelViscoacoustic(GenericModel):
+    """
+    The physical model used in seismic inversion processes.
+
+    Parameters
+    ----------
+    origin : tuple of floats
+        Origin of the model in m as a tuple in (x,y,z) order.
+    spacing : tuple of floats
+        Grid size in m as a Tuple in (x,y,z) order.
+    shape : tuple of int
+        Number of grid points size in (x,y,z) order.
```

Comment:
Number of grid points  - _size_ - in (x,y,z) order. (?)

Reply:
this is similar to pre-existing examples (e.g. acoustic, elastic ..)